source/RSRCFilter.cpp

```diff
@@ -0,0 +1,36 @@
+#include "RSRCFilter.h"
+
+#include <MimeType.h>
+#include <string.h>
+
+static const char* supported_types[] = {
+	"application/x-vnd.Be-elfexecutable",
```

Comment:
Normally,it's all lower case. As is (almost) all apps are filtered out... More testing before pushing, please. :)

Reply:
I was unsure what it was supposed to do... still getting my head around resource files in general.
I'll be more careful with pushing now! :)

Reply:
Allowing this mimetype allows for object (.o) files to be opened, which I assume is not what we want.
Should I add an additional check in the filter function explicitly rejecting object files?

Reply:
> Should I add an additional check in the filter function explicitly rejecting object files?

Sounds like a good idea to me. Just test for a ".o" suffix, I suppose?
BTW, there are a few stray tabs in RSRCFilter.cpp.